docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc

```diff
@@ -0,0 +1,81 @@
+
+
+== Prerequisites
+
+* Only available for Cloud and Hybrid customers.
+* Gather the internal IP address or hostname of the Mac mini host(s).
+* Kobiton administrator with the ADMIN predefined permission.
+* Remote update is available in deviceConnect version 4.13.0 and above.
+
+== Perform a remote deviceConnect update manually
+
+When the below notification banner appears in the Portal, a deviceConnect update is available for at least 1 Mac mini host in your organisation:
+
+image:new-version-available.png[width=1000,alt="The banner about new version of deviceConnect"]
+
+To update deviceConnect remotely, log into the Kobiton Portal with an administrator account.
+
+include::profile:partial$open-settings.adoc[]
+
+Select the *Device Management* tab.
+
+The *Device Management* page lists all dedicated mobile devices for the org grouped by Mac mini host. Locate the Mac mini host you want to update and select the corresponding *Upgrade* button.
+
+[TIP]
+A Mac mini host that does not have the latest deviceConnect version has a warning icon next to its name in Device Management.
+
+[IMPORTANT]
+Make sure the current deviceConnect version on the Mac mini host is at least 4.13.0 before continuing, as deviceConnect below this version cannot be remotely updated.
+
+image:upgrade-host-machine-device-management.png[width=1000,alt="The Upgrade button next to the hosting machine name in Device Management"]
+
+In the confirmation modal, choose *Upgrade* again to proceed.
+
+The Mac mini host will download the latest deviceConnect version and apply the update. Once the update starts, any active sessions on devices hosted by that Mac mini will be terminated. During the update, no actions can be performed on the Mac mini or the connected devices (all the buttons are grayed out).
+
+After the update finished, the devices should automatically come back online. Double-check the current version of deviceConnect on the Mac mini host to verify if the update is successful.
+
+== Enable or disable automatic deviceConnect update for an organization
+
+By default, automatic deviceConnect update is enabled for an organization. When automatic update is enabled, the system will periodically check for the latest deviceConnect version, download it, then update.
+
+The update only starts when all devices hosted by the Mac mini are not in use, i.e. there is no active sessions on the devices. If there are active sessions on the devices, the update is skipped until the next update check (about 30 minutes between each check).
+
+To check the current automatic update setting for your organization, or to enable/disable automatic update, follow the steps below:
+
+Log into the Kobiton Portal with an administrator account.
+
+include::profile:partial$open-settings.adoc[]
+
+Select the *Device Management* tab. If the organization has 1 least 1 Mac mini host with online devices, the *New version upgrade* section displays and *Automatic* is selected by default:
```

Comment:
Small typo - first 1 should be "at" - "If the organization has at least 1..."

Reply:
updated